### PR TITLE
Make boot output nicer

### PR DESCRIPTION
**Pull request by keszybz**

### CODE REVIEW

File: src/libsystemd/sd-bus/sd-bus.c [hunk 1]
```diff
@@ -4198,7 +4198,6 @@ _public_ int sd_bus_get_description(sd_bus *bus, const char **description) {
         assert_return(bus, -EINVAL);
         assert_return(bus = bus_resolve(bus), -ENOPKG);
         assert_return(description, -EINVAL);
-        assert_return(!bus_origin_changed(bus), -ECHILD);
```
Comment (bluca) [inline src/libsystemd/sd-bus/sd-bus.c:4202]: mmh, but is the object going to be valid? are we 100% sure?

Comment (keszybz) [reply src/libsystemd/sd-bus/sd-bus.c:4202]: A fork does not invalidate memory. (If it did, then even this check would be impossible.)  So the object is valid, we just don't want to use it.

Comment (bluca) [reply src/libsystemd/sd-bus/sd-bus.c:4202]: ok

**Comment by keszybz** [comment]
rpm-build:fedora-rawhide-i386 
  518/1195 systemd:test / test-time-util                                                                                        FAIL             1.94s   killed by signal 6 SIGABRT
Not related. I think we had an issue for this.

**Review by yuwata — APPROVED** [review]

**Pull request merged**